controller/api/destination/watcher/server_watcher.go

```diff
@@ -36,11 +44,23 @@ type ServerUpdateListener interface {
 	UpdateProtocol(bool)
 }
 
+var serverMetricsInst = serverMetrics{
+	subscribersGauge: promauto.NewGauge(prometheus.GaugeOpts{
+		Name: "server_subscribers",
+		Help: "Number of subscribers to any Server changes.",
+	}),
+	portsGauge: promauto.NewGauge(prometheus.GaugeOpts{
+		Name: "server_subscribers_port",
+		Help: "Number of pod ports targeted by Server subscribers.",
```

Comment:
So for each individual Server this will track how many subscribers we have for _any_ port in that Server, right? The idea is that we can correlate this with the global number?

Reply:
No, this doesn't track a subscribers count. It tracks the total number of ports that are being watched by subscribers. If say there's a pod for a service that is depended on by multiple subscribers downstream, `server_subscribers` will count all those subscribers but `server_subscribers_port` will only account for this single pod/port that is being targeted. Let me know if this makes sense :-)